### ISSUE TITLE
Allow us to force overwrite environment variables

### ISSUE THREAD

**Issue by agitelzon** [vercel/terraform-provider-vercel#193]
I currently have a workflow where I allow my developers to manually added environment variables via the Vercel website. Then later I add that environment variable to terraform for tracking and future changes. Because the env variable was manually created, I get an error of:
```
│ Could not create project environment variable, unexpected error: ENV_ALREADY_EXISTS - 
Another Environment Variable with the same Name and Environment exists in your project. 
Remove it or choose a different Name or Environment.
```

It would be nice if there was a parameter to tell the provider and API to overwrite or delete and recreate the current variable with what is in the terraform config.

I know that I can use terraform import, but importing environment variables is annoying because the environment variable id is needed and there is no easy way to see the ID in the website, only via the web browser development tools or API.

To recreate:
Create the vercel project.
```
resource "vercel_project" "test" {
  name = "test-foobar"
  git_repository = {
    type = "github"
    repo = "agitelzon/test"
  }
}
```
Manually create the environment variable `FOO` with any value via the web console.
Then add the following code and run it.
```
resource "vercel_project_environment_variable" "foo" {
  project_id = vercel_project.test.id
  key        = "FOO"
  value      = "foobar"
  target     = ["development", "preview", "production"]
}
```

**Comment by dglsparsons** [comment]
Hi @agitelzon, would it work if i made it spit out the ID of the existing resource, so it can be imported, in the case of their being a collision with an existing env var?

**Comment by agitelzon** [comment]
Thanks, @dglsparsons. Yes, anything would help.

**Comment by dglsparsons** [comment]
```
╷
│ Error: Error creating project environment variable
│
│   with vercel_project_environment_variable.test_2,
│   on main.tf line 25, in resource "vercel_project_environment_variable" "test_2":
│   25: resource "vercel_project_environment_variable" "test_2" {
│
│ Could not create project environment variable, unexpected error: ENV_ALREADY_EXISTS - Another Environment Variable with the same Name and Environment exists in your project. Remove it or choose a different Name or Environment. the conflicting environment variable ID is
│ prj_15Z0wAERpVw0h2qNi4tP9oOAyy2B/WURrO0mHStRZrUMn
╵
```

Here's an example of what this could look like 🙏

**Comment by agitelzon** [comment]
How difficult would it be to then automatically import that variable into terraform and modify it, if a parameter is set?

**Comment by dglsparsons** [comment]
You'd just copy the ID, and run `terraform import <resource path> <id>`
e.g. for the above, 
`terraform import vercel_project_environment_variable.test_2 prj_15Z0wAERpVw0h2qNi4tP9oOAyy2B/WURrO0mHStRZrUMn`. 

Then configure however you'd like.

**Comment by agitelzon** [comment]
Thanks, @dglsparsons! The manual import method would definitely work be a significant improvement, but it would be even better if we could automate the import of the manually created env variable directly within the provider. Maybe something like an `auto_import` or `force_overwrite` parameter? 

It would make things easier for anyone juggling manual and Terraform-managed env vars.

I appreciate the work you are doing on this and thank you for adding the comment field!

**Comment by dglsparsons** [comment]
Yeah... i don't want to add a `force_overwrite` or `auto_import` because that's encouraging some pretty dangerous practices and goes against how terraform is typically used. But I'll make it omit conflicting env var IDs so you can import manually.
